src/Catrobat/Controller/Web/CommentsController.php

```diff
@@ -111,4 +114,42 @@ public function postCommentAction(CatroNotificationService $notification_service
 
     return new Response(Response::HTTP_OK);
   }
+
+  /**
+   * @Route("/translateComment", name="translate_comment", methods={"GET"})
+   */
+  public function translateCommentAction(TranslationDelegate $translationDelegate): Response
+  {
+    $em = $this->getDoctrine()->getManager();
+    $comment = $em->getRepository(UserComment::class)->find($_GET['CommentId']);
+
+    if (null === $comment) {
+      return new Response('No comment found for this id '.$_GET['CommentId'], Response::HTTP_BAD_REQUEST);
+    }
+
+    $source_language = $_GET['SourceLanguage'] ?? null;
```

Comment:
use $request with query get instead of $_GET for some internal validations in the background